ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ExternalBackupIntegrations/icons.tsx

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable no-void */
+import React, { ReactElement } from 'react';
+import { Popover } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+export function gcsWorkloadIdentity(): ReactElement {
```

Comment:
@stehessel Since this function returns elements, I'd recommend creating a React `Component` instead. Usually, we will use `functions` or `hooks` for reusable `logic` while we use `components` for reusable `visual elements`.

Reply:
Both of these look very similar, so I recommend creating one reusable component and using it for both cases, distinguishing them based on the props we pass to the component.

For this purpose, I would create an `IntegrationHelpIcon` component. The name clearly describes the functionality and is generic enough to work in different scenarios.

To decide what props to use, let's examine the similarities and differences. The `href` and `text` in the `bodyContent` differ between the two cases. It seems like `href` is applied to the entire body, which we should avoid. Instead, we want it on a specific word or phrase in the text. In this case, we could put it on "See the Red Hat RHACS documentation."

Here’s a breakdown of the props:

**helpTitle**: The `headerContent` differs, so this prop will handle the `title`.
**helpText**: This prop will contain the body text, including the linked phrase.
**ariaLabel**: This prop will make the component accessible with unique labels for different cases.
We can remove `aria-describedby` since it's using static text that doesn’t describe anything specific at the moment.

So, we will have an `IntegrationHelpIcon` component with the props: `helpTitle`, `helpText`, and `ariaLabel`. This approach is clean, simple, and straightforward. I'd place this component in a separate file within `Forms/ExternalBackupIntegrations` or one level up if we plan to use it in other parts of the Integrations screens.

Example of usage:

```
<IntegrationHelpIcon
      helpTitle="GCP workload identity"
      helpText={
          <div>Enables authentication via short-lived tokens using GCP workload identities. <a href="https://docs.openshift.com/acs/integration/integrate-using-short-lived-tokens.html" target="_blank" rel="noreferrer">See the Red Hat RHACS documentation</a> for more information.</div>
      }
      ariaLabel="Help for short-lived tokens"
/>
```

Reply:
@sachaudh thanks for your help. I've extracted the visuals into a new component as you suggested.